_data/team.yml

```
@@ -256,3 +256,11 @@
   username: shiqi.mei
   photo: no
   country: China
+  
+- name: Eduardo Pereira
+  role: Head of Sales North America
+  department: Sales
+  github_username: epereira2020
+  username: Edu Pereira
```

```suggestion
  username: eduardo.pereira
```

This needs to be your exact username on open.rocket.chat